customer-terminate-service/app/src/main/java/se/inera/intyg/cts/application/api/EraseController.java

```diff
@@ -0,0 +1,27 @@
+package se.inera.intyg.cts.application.api;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import se.inera.intyg.cts.application.service.EraseService;
+
+@RestController
+@RequestMapping("/api/v1/erase")
+public class EraseController {
+
+  private Logger LOG = LoggerFactory.getLogger(EraseController.class);
+  
+  private final EraseService eraseService;
+
+  public EraseController(EraseService eraseService) {
+    this.eraseService = eraseService;
+  }
+
+  @PostMapping("")
+  void startErase() {
+    LOG.info("Start erase");
```

Comment:
The endpoint is just a way to trigger 'Erase' instead of it being started by the EraseCareProviderTask. It will call the EraseService.erase() that will process/erase any termination that has the correct status.